clang/unittests/Format/TokenAnnotatorTest.cpp

```diff
@@ -3538,6 +3538,11 @@ TEST_F(TokenAnnotatorTest, TemplateInstantiation) {
   ASSERT_EQ(Tokens.size(), 11u) << Tokens;
   EXPECT_TOKEN(Tokens[2], tok::less, TT_TemplateOpener);
   EXPECT_TOKEN(Tokens[6], tok::greater, TT_TemplateCloser);
+
+  Tokens = annotate("return std::conditional_t<T::value == U::value, T, U>{};");
+  ASSERT_EQ(Tokens.size(), 21u) << Tokens;
+  EXPECT_TOKEN(Tokens[4], tok::less, TT_TemplateOpener);
+  EXPECT_TOKEN(Tokens[16], tok::greater, TT_TemplateCloser);
```

Comment:
Will this also verify that there is no extra space between `>` and `{`? I didn't mention it in the ticket but it was another issue.

Reply:
The root cause was the misannotation of the `>`, so we add an annotation test instead of verifying the spacing directly.

Reply:
Ok, thanks for the clarification!